x-pack/solutions/security/plugins/security_solution/common/constants.ts

```diff
@@ -521,3 +521,16 @@ export const MAX_MANUAL_RULE_RUN_BULK_SIZE = 100;
  * Whether it is a Jest environment
  */
 export const JEST_ENVIRONMENT = typeof jest !== 'undefined';
+
+export const AI_FOR_SOC_INTEGRATIONS = [
```

Comment:
This constant actualy already in a shared folder https://github.com/elastic/kibana/blob/main/x-pack/solutions/security/plugins/security_solution/public/common/lib/search_ai_lake/integrations.ts#L9